### DISCUSSION TITLE
Ajout de document à la réservation

### DISCUSSION THREAD

**Discussion by Bernidouille** [JeromeDevome/GRR#389] [Idées]
Bonjour,

Serait-il envisageable d'ajouter un ou plusieurs documents (pièce jointe) à la réservation?
Via le système de champs additionnels avec un type "File".
Le stockage pourrais être géré de la même manière que les images des ressources?
Si cette fonctionnalité vous semble utile, soutenez ma demande.
Merci,
B.

**Comment by taib27** [comment]
Bonjour, 
Toujours pas de possibilité d'ajout de PJ sur les demande de réservation ?!

Merci par avance pour tout vos efforts et le travaille accompli . 

BIG UP ^_^

**Reply by ynaessens** [reply]
Bonjour,
je reviens tardivement sur ce fil : l'ajout de documents est possible en version 3.5.2.
Cordialement
YN

**Comment by JeromeDevome** [comment]
Bonjour, cela est disponible dans la version 4.5
